src/lib.rs

```diff
@@ -72,7 +72,7 @@ use std::ffi::{OsStr, OsString};
 use std::fmt;
 use std::io;
 use std::ops::{Bound, RangeBounds};
-use std::path::{PathBuf, Path};
```

Comment:
`Path` is actually still in use:
https://github.com/rust-lang/pkg-config-rs/blob/dc20226158dbf9d67fe6eb469ab1899edfcd804c/src/lib.rs#L636
Hence I think we should leave it in or refactor that assertion to use `PathBuf`.
`cargo test` fails for me on macOS so I'm not sure how the CI build succeeded.

Reply:
Indeed

Reply:
Not relevant anymore with latest master.